stable/appmesh-manager/ci/values.yaml

```diff
@@ -0,0 +1,7 @@
+# CI testing values for appmesh-manager
+
+region: us-west-2
+image:
+  repository: stefanprodan/aws-app-mesh-controller-for-k8s
```

Comment:
A v1beta2 image should be pushed to Docker Hub and used here for the smoke test.

Reply:
ok